src/process_args.c

```diff
@@ -89,12 +89,31 @@ process_args(nmsgtool_ctx *c) {
 		usage(NULL);
 
 	if (c->version) {
-#ifdef HAVE_LIBZMQ
-		fprintf(stderr, "%s: version %s\n", argv_program, PACKAGE_VERSION);
-#else /* HAVE_LIBZMQ */
-		fprintf(stderr, "%s: version %s (without libzmq support)\n",
-			argv_program, PACKAGE_VERSION);
+		int support = 0;
+		fprintf(stderr, "%s: version %s", argv_program, PACKAGE_VERSION);
+#ifndef HAVE_LIBZMQ
+		support |= 1;
```

Comment:
Might be easier to understand with explicit bit math:
`
support |= 0x01;
...
support |= 0x10;
...
case 0x01:
...
case 0x10:
...
case 0x11:
`